geotrek/feedback/tests/test_views.py

```diff
@@ -79,7 +114,7 @@ def get_expected_json_attrs(self):
             'comment': self.obj.comment,
             'related_trek': None,
             'email': self.obj.email,
-            'status': None,
+            'status': self.obj.status_pk,
```

Comment:
```suggestion
            'status': self.obj.status_id,
```

Reply:
PK is an alias to primary key id by default. when need this ID by using foreign key, use <foreign key object>_id